fbpic/fields/fields.py

```diff
@@ -549,11 +647,15 @@ def filter_spect( self, fieldtype ) :
         ---------
         fieldtype : string
             A string which represents the kind of field to be filtered
-            (either 'E', 'B', 'J', 'rho_next' or 'rho_prev')
+            (either 'E', 'B', 'J', 'rho_next', 'rho_prev' or 'A')
```

Comment:
For now, since we do not need the `filter` method yet, I would recommend to revert it to its previous version (without A)